ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -604,7 +823,7 @@ class EclCmdRun : public EclCmdWithEclTarget
             req->setCluster(wuCluster.str());
         else if (optTargetCluster.length())
             req->setCluster(optTargetCluster.get());
-        req->setWait((int)optWaitTime);
+        req->setWait((int)(optPoll ? 0 : optWaitTime));
```

Comment:
I was thinking that even in poll mode, you would wait for (say) 1 second the first time you issue the call, so that subsecond executions did not incur any additional delay. Not a massive deal if that's too complex, but...